packages/ibm-products/src/components/Datagrid/Datagrid/DatagridRow.tsx

```diff
@@ -11,7 +11,7 @@ import { px } from '@carbon/layout';
 import { selectionColumnId } from '../common-column-ids';
 import cx from 'classnames';
 import { pkg, carbon } from '../../../settings';
-import { DatagridSlug } from './addons/Slug/DatagridSlug';
+import { DatagridAILabel } from './addons/AILAbel/DatagridLabel';
```

Comment:
can fix the naming once we get a clarity on this

Reply:
Thanks @devadula-nandan.
This is updated.